### PR TITLE
add python version when running pip env in README

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Open a browser at http://\<server-address\>:8880 to view and control the game.
 
 Open a pipenv shell that you'll use for the following commands:
 
-    pipenv shell
+     pipenv --python /usr/local/bin/python3.7 shell
```
Comment (sleviim) [inline README.md:81]: redundant space at the beginning

File: README.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ To install development requirements:
 
 To open a shell for development, use:
 
-    pipenv shell
+     pipenv --python /usr/local/bin/python3.7 shell
```
Comment (sleviim) [inline README.md:130]: same

**Review by sleviim — CHANGES_REQUESTED** [review]
I thought the machines have python3 installed

**Comment by shiramax** [comment]
@sleviim changing the default version cased a lot of issues in yum command, Ilana and I agree this is a better solution.

**Review by sleviim — APPROVED** [review]

**Comment by shiramax** [comment]
@sleviim can you merge this one?

**Pull request merged**